testsuite/integration/pom.xml

```diff
@@ -198,15 +196,6 @@
             </modules>
         </profile>
 
-        <!-- -Dts.picketlink -->
```

Comment:
Do you plan to remove elytron profile from TS in this MR? AFAIU Elytron would be default (and only one) security, so it may be better to make current behaviour from elytron profile default.

```
 <profile>
            <id>elytron.profile</id>
```